### ISSUE TITLE
Add-in "steals" focus from the compose window onload

### ISSUE THREAD

**Issue by austinwang** [OfficeDev/office-js#1683]
The addin seems to takes the "focus" away from the compose window when it loads.
Example use case is this.
1. User has our Addin "pinned" so that it auto-opens in new windows
1. User clicks reply to an email
1. Outlook opens reply window and user starts typing their reply
1. Outlook loads the our addin in the side of the compose window (this takes a couple seconds) - On load, the focus is taken out of the compose text box where the user was typing and so the user is still typing but no more text is appearing.
User is confused and thinks that our is actually "freezing outlook" and doesn't know to click back into the compose text box

We tested this with 1-2 other addins and it seems to be a consistent issue. Just curious if there's a way around this or if it is a known issue? It would be nice if the addin didn't take focus away from what the user is doing since the Addin is very rarely going to be the immediate / initial thing you interact with upon opening a new window. It's usually there to show context / take supplemental actions upon a given email.

## Expected Behavior
Open an reply window with a pinned addin, immediately start typing. The addin on the side should fully load without stealing focus away from the current focus which is the compose window body.

## Current Behavior
It DOES steal focus so user needs to wait 1-3 seconds before starting to type their reply, OR click back into the text area to restore focus there.

## Steps to Reproduce, or Live Example
1. User has our Addin "pinned" so that it auto-opens in new windows
1. User clicks reply to an email
1. Outlook opens reply window and user starts typing their reply
1. Outlook loads the our addin in the side of the compose window (this takes a couple seconds) - On load, the focus is taken out of the compose text box where the user was typing and so the user is still typing but no more text is appearing.
User is confused and thinks that our is actually "freezing outlook" and doesn't know to click back into the compose text box

## Context
It would be nice if the addin didn't take focus away from what the user is doing since the Addin is very rarely going to be the immediate / initial thing you interact with upon opening a new window. It's usually there to show context / take supplemental actions upon a given email.

## Your Environment
This seems to affect outlook desktop on mac and pc

**Comment by AlexJerabek** [comment]
Thanks for raising this concern @austinwang. 

@exextoc, could you please weigh in on the intent of this behavior and if it's something that can be changed? Thanks!

**Comment by exextoc** [comment]
On Win32. What build number are you on? Talking with the developer team, part of this was fixed in build 16.0.13231.10000. 
However, there are certain situations where this bug can still repro, and that bug is still in the backlog. I don't have a timetable for that bug at this time. 

For Mac, I've contacted the Mac team about this. I/they may reply with more questions in the future. 

Internal Tracking Id: Office: 4387430 & 4155408

**Comment by exextoc** [comment]
@austinwang The issue is reprodcible in only on old UI on MAC. In new UI it works fine for me. Can you please try the experience on new UI?

**Comment by austinwang** [comment]
Thanks for the quick response. We're following up to get the build version and will let u know what we were seeing it on.

**Comment by Sageparker47** [comment]
@exextoc Here is the build number that our customer is on.
<img width="284" alt="Zoom_Meeting" src="https://user-images.githubusercontent.com/27702295/110394793-d8880100-8021-11eb-800f-45646a01ec51.png">

We were also asked if "64-bit" would play a factor in this?

**Comment by exextoc** [comment]
As mentioned above, for Win32 the primary fix is in build 16.0.13231.10000 and above. Please update your Outlook for the fix in Windows.

**Comment by splinter** [comment]
We have a customer who is experiencing the issue with build `13628.20448`. They are on a Semi Annual release channel.

The steps to reproduce the issue are identical. On this system the `Evernote` add-in does not cause the issue to reproduce.

EDIT: Just putting this information here to help the debugging process

**Comment by exextoc** [comment]
Thank you for providing additional build information. We are aware of another issue in win-32 that causes some pinned add-ins to take focus on compose. We have shared this information with the appropriate team, and it has been put on the backlog. We unfortunately have no timelines to share at this point.

Internal tracking id: Office: 4387430

**Comment by zacfarnsworth** [comment]
@austinwang Are you still experiencing this issue? I am trying to reproduce it on my machine but it appears to not reproduce.
